README.md

```diff
@@ -1 +1,245 @@
+
+That front panel with all of the buttons is its *interface*—the public presentation of all the ways that the object is intended to be used. Its *implementation*, however, is kept privately enclosed in a metal box that slides into the dashboard. This is partly for safety (it would be awful to electrocute a passenger!), but mainly for security: the radio manufacturer doesn't want the internal circuitry to get disturbed! That might cause unexpected behavior. 
+
+Providing an interface about public usability via the `.h` header file is how Objective-C classes share information about themselves to other classes. However, *within* that class file's `.m` implementation file, an optional *private interface* can be used for declaring *private properties*. 
+
+The private `@interface` is placed in the `.m` implementation file between the `#import` lines and the `@implementation` section and must be closed with an `@end` keyword before the `@implementation` keyword. Private properties can be added here without making them visible outside of the class. Private *methods*, however, don't need to be declared in the private interface section—the method names in the `@implementation` section make them visible within the current class file.
+
+So, an simple representation of that radio's interface in Objective-C might look something like this:
+
+```objc
+//  FISRadio.h
+
+#import <Foundation/Foundation.h>
+#import <CoreGraphics/CGBase.h>   // imports CGFloat
+
+@interface FISRadio : NSObject
+
+@property (nonatomic) CGFloat amFrequency;
+@property (nonatomic) CGFloat fmFrequency;
+
+@property (nonatomic) CGFloat swBal;
+@property (nonatomic) CGFloat tuning;
+
+@property (nonatomic) CGFloat tone;
+@property (nonatomic) CGFloat volume;
+
+@property (nonatomic) BOOL stereo;
+
+@property (nonatomic) CGFloat fmPreset1;
+@property (nonatomic) CGFloat fmPreset2;
+@property (nonatomic) CGFloat amPreset1;
+@property (nonatomic) CGFloat amPreset2;
+@property (nonatomic) CGFloat amPreset3;
+
+@end
+```
+
+## The Anatomy Of A Property
+
+Using the `@property` syntax, while it can feel verbose, is actually  a shorthand for generating the three components of a property: the instance variable, the "setter" method, and the "getter" method.
+
+#### The Instance Variable
+
+The **instance variable** (also "ivar") is the actual value that the property manages. It is both standard convention and implicit syntax that the instance variable is name after the property, but prepends an underscore character (`_`) for disambiguation. So, `_length` is the *instance variable* managed by the `length` property, and `_count` is the *instance variable* managed by the `count` property.
+
+**Note:** *Do* ***not*** *access an instance variable directly unless writing an initializer (common) or overriding a setter or getter (advanced). It best practice to use dot notation which implicitly calls the setter and getter methods for that property.*
```

Comment:
I would be even stronger than "it's best practice...". "Use dot notation (which implicitly calls the setter and getter methods) everywhere else."